src/cascadia/TerminalControl/ControlInteractivity.cpp

```diff
@@ -182,7 +182,12 @@ namespace winrt::Microsoft::Terminal::Control::implementation
         }
         else if (_canSendVTMouseInput(modifiers))
         {
-            _core->SendMouseEvent(terminalPosition, pointerUpdateKind, modifiers, 0, buttonState);
+            auto adjustment = 0;
+            if (_core->ScrollOffset() > 0)
+            {
+                adjustment = _core->BufferHeight() - _core->ScrollOffset() - _core->ViewHeight();
+            }

```

Comment:
```suggestion
            const auto adjustment = _core->ScrollOffset() > 0 ? _core->BufferHeight() - _core->ScrollOffset() - _core->ViewHeight() : 
                                                                0;
```
If you use a ternary operator, you can throw a `const` in there

Reply:
Alternatively you can drop the `const` on `terminalPosition` and just modify the Y value in place within the if condition.